docs/static/advanced-pipeline.asciidoc

```diff
@@ -161,18 +161,20 @@ If your pipeline is working correctly, you should see a series of events like th
 [source,json]
 --------------------------------------------------------------------------------
 {
-    "@timestamp" => 2016-10-11T20:54:06.733Z,
+    "@timestamp" => 2017-11-09T01:44:20.071Z,
         "offset" => 325,
       "@version" => "1",
           "beat" => {
+            "name" => "My-MacBook-Pro.local",
         "hostname" => "My-MacBook-Pro.local",
-            "name" => "My-MacBook-Pro.local"
+         "version" => "6.0.0"
     },
-    "type" => "log",
           "host" => "My-MacBook-Pro.local",
+    "prospector" => {
+        "type" => "log"
```

Comment:
@dedemorton my point here was, didn't Beats, in general, start defaulting type => doc instead of log in 6.0? https://github.com/elastic/beats/pull/3757

Reply:
Unless the prospector "type" is different from the default "doc type". You may be understand it better than I do.

Reply:
Yes, that's the difference. If you look at the full output, you'll see that the `_type` field is set to `doc` as expected. The `type` filed on line 174 is the prospector type.

Reply:
Makes sense, thanks!